packages/babel-core/src/config/validation/options.js

```diff
@@ -216,6 +243,12 @@ export function validate(type: OptionsType, opts: {}): ValidatedOptions {
     if (type === "env" && key === "env") {
       throw new Error(`.${key} is not allowed inside another env block`);
     }
+    if (type === "env" && key === "overrides") {
+      throw new Error(`.${key} is not allowed inside an env block`);
+    }
+    if (type === "override" && key === "overrides") {
+      throw new Error(`.${key} is not allowed inside an overrides block`);
```

Comment:
Nit: You can write `throw new Error(".overrides is not allowed inside an overrides block");`, since `key` is guaranteed to be `overrides` because of the line above.

Reply:
Yeah I just felt like it was more consistent this way.